docs/architecture/adr-053-state-sync-prototype.md

```diff
@@ -64,11 +72,13 @@ message ResponseListSnapshots {
 // Offers a snapshot to the application
 message RequestOfferSnapshot {
     Snapshot snapshot = 1;
+    bytes app_hash = 2;
 }
 
 message ResponseOfferSnapshot {
     bool accepted = 1;
-    enum reason {             // Reason why snapshot was rejected
+    Reason reason = 2;        // Reason why snapshot was rejected
+    enum Reason {
```

Comment:
minor nit: the scoping rules of protobuf dictate that enum constant values must be unique within a protobuf file, hence calling it UNKNOWN_CHANGE_TYPE and not UNKNOWN.

Reply:
I believe this only applies to sibling enums. So the `unknown` enum values in `ResponseOfferSnapshot` and `ResponseApplySnapshotChunk` are not in conflict, because they are not siblings. Or were you thinking about possible conflicts with future additional enums within the same response classes?